docs/topics/wasm/wasm-get-started.md

```diff
@@ -9,33 +9,38 @@ app with Kotlin/Wasm in IntelliJ IDEA, and generate artifacts to publish as a si
 
 ## Before you start
 
-1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
-2. Clone the [Kotlin/Wasm examples](https://github.com/Kotlin/kotlin-wasm-examples/tree/main) repository 
-  by selecting **File** | **New** | **Project from Version Control** in IntelliJ IDEA.
-
-   You can also clone it from the command line:
+Create a project using the Kotlin Multiplatform wizard:
 
-   ```bash
-   git clone git@github.com:Kotlin/kotlin-wasm-examples.git
-   ```
-   
-> Alternatively, you can use our experimental [web wizard](https://kmp.jetbrains.com/) to download a sample project.
+> The Kotlin Multiplatform wizard is an [Experimental](components-stability.md) feature. It may be changed at any time.
 >
 {type="note"}
 
+1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com/#newProject).
+2. On the **New Project** tab, change the project name to "ComposeDemo" and the project ID to "kmp.project.demo".
```

Comment:
If we're guiding the user toward a purely Wasm project, the names here could reflect that. But it's just a suggestion :)